projectile.el

```diff
@@ -417,6 +417,11 @@ Any function that does not take arguments will do."
   :type 'hook
   :package-version '(projectile . "0.14.0"))
 
+(defcustom projectile-ag-use-ignored t
+  "If true, use ignored paths/patterns in search."
```

Comment:
Probably you should expand this docstring to mention that we're talking about projectile's ignores and not VC ignores.